sdcm/utils/sstable/s3_uploader.py

```
@@ -9,6 +9,76 @@
+    Returns:
+        Tuple of (S3 download link, S3 filename) or ("", "") on failure
+    """
+    LOGGER.info("Collecting %s table from node %s...", table_name, node.name)
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+    filename = f"sct_{table_name.replace('.', '_')}_{timestamp}.jsonl"
+    remote_file = f"/tmp/{filename}"
+    s3_filename = f"{table_name.replace('.', '_')}-{timestamp}-{node.name}.jsonl.tar.gz"
+
+    try:
+        cqlsh_cmd = node._gen_cqlsh_cmd(
+            command=f"SELECT JSON * FROM {table_name}",
```

Ruff (0.15.14)

[error] Possible SQL injection vector through string-based query construction

(S608)